### PR TITLE
rmw 0.9.1

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/rmw.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Rmw < Formula
   desc "Safe-remove utility for the command-line"
```
Comment (andy5995) [inline Formula/rmw.rb:2]:
```suggestion
  desc "Trashcan/recycle bin utility for the command-line"
```

File: Formula/rmw.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
   homepage "https://remove-to-waste.info/"
```
Comment (andy5995) [inline Formula/rmw.rb:3]:
```suggestion
  homepage "https://theimpossibleastronaut.github.io/rmw-website/"
```

**Review by andy5995 — COMMENTED** [review]
The automatic changes LGTM! I'm the rmw maintainer. These two changes I'm suggesting I can do in a separate PR, if you prefer, after this is merged as-is.

File: Formula/rmw.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/rmw.rb]: 👍

File: Formula/rmw.rb (hunk 2 above)
Comment (chenrui333) [reply Formula/rmw.rb]: 👍

File: Formula/rmw.rb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 class Rmw < Formula
-  desc "Safe-remove utility for the command-line"
-  homepage "https://remove-to-waste.info/"
-  url "https://github.com/theimpossibleastronaut/rmw/releases/download/v0.9.0/rmw-0.9.0.tar.xz"
-  sha256 "cc9d20733c9f9945054041ee6aeac7f4a4b7a675f297ffe388e4863fb84ed4a1"
+  esc "Trashcan/recycle bin utility for the command-line"
```
Comment (alebcay) [inline Formula/rmw.rb:2]:
```suggestion
  desc "Trashcan/recycle bin utility for the command-line"
```

**Review by p-linnane — APPROVED** [review]

**Pull request merged**